main.go

```diff
@@ -36,6 +37,8 @@ func main() {
 	startCallbacker := flag.Bool("callbacker", false, "start callbacker")
 	useTracer := flag.Bool("tracer", false, "start tracer")
 	help := flag.Bool("help", false, "Show help")
+	var regtest bool
+	flag.BoolVar(&regtest, "regtest", false, "Use regtest-config.yml instead of the default config.yaml")
```

Comment:
I would suggest here to allow giving the name of the config file optionally. That would be more flexible.

`
config := flag.String("configFile", "config", "name of the config yaml file")
`